packages/usa-in-page-navigation/src/index.js

```diff
@@ -14,13 +14,14 @@ const IN_PAGE_NAV_ROOT_MARGIN = "0px 0px 0px 0px";
 const IN_PAGE_NAV_THRESHOLD = "1";
 const IN_PAGE_NAV_CLASS = `${PREFIX}-in-page-nav`;
 const IN_PAGE_NAV_ANCHOR_CLASS = `${PREFIX}-anchor`;
+const IN_PAGE_NAV_CONTAINER_CLASS = `${IN_PAGE_NAV_CLASS}__container`;
+const IN_PAGE_NAV_MAIN_CLASS = `${IN_PAGE_NAV_CLASS}__main`;
 const IN_PAGE_NAV_NAV_CLASS = `${IN_PAGE_NAV_CLASS}__nav`;
 const IN_PAGE_NAV_LIST_CLASS = `${IN_PAGE_NAV_CLASS}__list`;
 const IN_PAGE_NAV_ITEM_CLASS = `${IN_PAGE_NAV_CLASS}__item`;
 const IN_PAGE_NAV_LINK_CLASS = `${IN_PAGE_NAV_CLASS}__link`;
 const IN_PAGE_NAV_TITLE_CLASS = `${IN_PAGE_NAV_CLASS}__heading`;
 const SUB_ITEM_CLASS = `${IN_PAGE_NAV_ITEM_CLASS}--sub-item`;
-const MAIN_ELEMENT = "main";
```

Comment:
I'm not a maintainer, just a fellow contributor, but: Since this would be a breaking change as proposed, would another option be to support an optional attribute on the In-Page Navigation element to override the selector for the main element, so you could do something like ... ?

```html
<aside
  class="usa-in-page-nav__container"
  data-main-selector=".usa-in-page-nav__main"
></aside>
```

Or at least, could backward-compatibility be maintained by continuing to support the `main` element, if present?